extension/extension.ts

```diff
@@ -13,16 +13,17 @@ import { SnapWindowExtension } from './src/snapWindow';
 import * as DBusUtils from './src/utils/dbus';
 import * as VKeyboard from './src/utils/keyboard';
 
-const ExtensionUtils = imports.misc.extensionUtils;
+const extensionObject = Extension.lookupByUUID('gestureImprovements@gestures');
 
-class Extension {
+class GNOMEGestureImprovementsExtension extends Extension {
```

Comment:
export default class GnomeGestureImprovements extends  Extension {

line 156 is not necessary??